server/restivus-swagger.js

```diff
@@ -32,8 +32,11 @@ Restivus.prototype.addSwagger = function(path) {
           if(route.path !== path &&
             route.path !== 'login' && route.path !== 'logout' )
           {
+            // Modify path parameter to swagger spec style
+            // Replaces :param with {param}
+            let newpath = route.path.replace(/:(\w+)/g, '{$1}');
             // Use path as key
-            let key = '/'.concat(route.path);
+            let key = '/'.concat(newpath);
```

Comment:
If this variable will not be reassigned, use [`const`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) instead of `let`.